extensions/mssql/package.nls.json

```diff
@@ -79,6 +79,7 @@
 	"mssql.connectionOptions.authType.categoryValues.sqlLogin": "SQL Login",
 	"mssql.connectionOptions.authType.categoryValues.integrated": "Windows Authentication",
 	"mssql.connectionOptions.authType.categoryValues.azureMFA": "Azure Active Directory - Universal with MFA support",
+	"mssql.connectionOptions.authType.categoryValues.activeDirectoryPassword": "Active Directory - Password",
```

Comment:
I guess this is the "best" name we have for this option? 😄

Reply:
I really want a better name! Have any suggestions?